stripe/src/test/java/com/stripe/android/paymentsheet/CurrencyFormatterTest.kt

```diff
@@ -225,23 +224,6 @@ class CurrencyFormatterTest {
             .isEqualTo("US$ 1,234.12")
     }
 
-    /**
-     * This test would be good to run on android phones to see where the apparent inconsistency
-     * with the server is coming from (See `CurrencyFormatter.SERVER_DECIMAL_DIGITS_2`).
-     */
-    @Test
-    fun `Currencies don't match server value of 2`() {
-        val non2Value = CurrencyFormatter.SERVER_DECIMAL_DIGITS_2
-        Currency.getAvailableCurrencies()
-            .filter { non2Value.contains(it.currencyCode) && it.defaultFractionDigits != 2 }
-            .sortedBy {
-                it.currencyCode
-            }
-            .forEach {
-                println("Found a currency that does not have 2 decimal places: " + it.currencyCode)
-            }
-    }
```

Comment:
If you want to keep this for posterity, you can annotate it with `@Ignore` instead of `@Test`